index.html

```diff
@@ -43,26 +43,26 @@
               <a class="nav-link" href="about.html">About</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="services.html">Services</a>
+              <a class="nav-link" href="login.html">Log In</a>
```

Comment:
I added a "create an account" link in index.html but maybe having login.html serve both purposes would be better.